docker-compose.yml

```diff
@@ -1,4 +1,4 @@
-version: "3.1"
+version: "3.2"
```

Comment:
as you have the same version in base and override files, why redefine it in the overrides?

Reply:
This is the version of the config file. I believe it is a required parameter.
Docker requires that it be the same across base and override files.

Reply:
good to know - I was confusing this with an instruction or tag field, rather than versioning of the config file itself.